dockerfiles/framework/scale/deployDb.py

```diff
@@ -25,8 +28,7 @@ def run():
     marathon = '''{"id": "/%(scaleDBHost)s",
                  "instances": 1,
                  "constraints": [["hostname","UNIQUE"]],
-                 "container": {"docker": {"forcePullImage": true,
```

Comment:
Keeping it there slows down deployment by ignoring the local image cache. It needs to pull the image again every time you startup.